client/src/components/overview/overview-components/Stars.jsx

```diff
@@ -1,15 +1,25 @@
 import styled from 'styled-components';
 import React, { useState } from "react";
+import StarRatings from 'react-star-ratings';
 
 const StyledOverviewStars = styled.div`
+padding: 10px;
 grid-area: OvStar;
 margin-top: 50px
 `;
 
-export default function OverviewStars({ stars }) {
+export default function OverviewStars({ stars, reviewCount }) {
   return (
-    <StyledOverviewStars>
-        <p>&#9733;&#9733;&#9733;&#9733;&#9733;</p>
+    <StyledOverviewStars data-testid="stars">
+        <StarRatings
```

Comment:
what is parseFloat?

Reply:
Parse float leaves the decimals in. ParseInt removes everything after the dot.